lib/init.js

```diff
@@ -55,11 +68,21 @@ const isMixProject = async (filePath) => {
   return existsSync(join(project, 'mix.exs'));
 };
 
+const isUmbrellaProject = async (filePath) => {
+  const project = await elixirProjectPath(filePath);
+  return existsSync(join(project, 'apps'));
+}
+
 const isTestFile = async (filePath) => {
   const project = await elixirProjectPath(filePath);
   const relativePath = relative(project, filePath);
-  // Is the first directory of the relative path "test"?
-  return relativePath.split(sep)[0] === 'test';
+  if (isUmbrellaProject()) {
+    // Is the structure "apps/app_name/test/..."
+    return relativePath.split(sep)[2] === 'test';
+  } else {
```

Comment:
Since the function will `return` anyway if the conditional is true, you can just end the `if` and leave the following statements out of it.

So this would look like:

```js
  if (isUmbrellaProject()) {
    // Is the structure "apps/app_name/test/..."
    return relativePath.split(sep)[2] === 'test';
  }
  // Is the structure "test/..."
  return relativePath.split(sep)[0] === 'test';
```